src/custom-elements/stencil.config.ts

```diff
@@ -25,6 +26,52 @@ export const config: Config = {
     {
       type: 'dist'
     },
+    {
+      type: 'docs-custom',
+      generator: (docs: JsonDocs) => {
+        const alertTitleProp = {
+          name: 'title',
+          type: 'string',
+          mutable: false,
+          attr: 'title',
+          reflectToAttr: true,
+          docs: 'to define alert title.',
+          docsTags: [],
+          default: 'false',
+          values: [
+            {
+              type: 'string'
+            }
+          ],
+          optional: false,
+          required: false
+        };
+
+        const popoverTitleProp = {
+          name: 'title',
+          type: 'string',
+          mutable: false,
+          attr: 'title',
+          reflectToAttr: true,
+          docs: 'to define popover title.',
+          docsTags: [],
+          default: 'false',
+          values: [
+            {
+              type: 'string'
+            }
+          ],
+          optional: false,
+          required: false
+        };
+
+        docs['components'].find(x => x.tag === 'chi-alert')['props'].push(alertTitleProp);
+        docs['components'].find(x => x.tag === 'chi-alert')['props'].sort((a, b) => (a.name > b.name) ? 1 : -1);
```

Comment:
Please return 0 when a.name equals b.name. Depending on sort implementation it might fail.

Reply:
Done, thanks!